src/Files/ValueObjects/MimeType.php

```diff
@@ -169,6 +169,21 @@ public static function isValid(string $mimeType): bool
         );
     }
 
+    /**
+     * Checks if this MIME type has a specific MIME type.
+     *
+     * This method checks if the MIME type starts with the specified MIME type.
+     *
+     * @since n.e.x.t
+     *
+     * @param string $mimeType The MIME type to check.
+     * @return bool True if the MIME type has the specified MIME type.
+     */
+    public function hasMimeType(string $mimeType): bool
+    {
+        return strpos($this->value, $mimeType . '/') === 0;
+    }
```

Comment:
The VO is intended to indicate a single value, so it's more accurate to say that the value _is_ a certain type rather than "has" a type.

Also, the internal value is stored as lowercase, so let's make sure the `$mimeType` is also made lowercase. Good to add this to the tests, too.
```suggestion
    /**
     * Checks if this MIME type is a specific MIME type, the type being the prefixed part (e.g. "audio/").
     *
     * @since n.e.x.t
     *
     * @param string $mimeType The MIME type to check.
     * @return bool True if the MIME type is the specified MIME type.
     */
    public function isType(string $mimeType): bool
    {
        return strpos($this->value, strtolower($mimeType) . '/') === 0;
    }
```

Reply:
Please note that I didn't adjust this in all other places.

Reply:
Thanks for the review, @JasonTheAdams 

✅ Renamed hasMimeType() to isType().
✅ Added strtolower() for the argument, and updated the test.

Let me know if you'd like any other tweaks!